kafka-connect-http/pom.xml

```diff
@@ -11,6 +11,10 @@
     <artifactId>kafka-connect-http</artifactId>
     <name>Kafka Connect HTTP</name>
 
+    <properties>
+        <env.GPG_PASSPHRASE>bogus</env.GPG_PASSPHRASE>
+    </properties>
```

Comment:
Out of curiosity, what did you need this for?

Reply:
This was just to avoid the error message from the pom syntax checker.
I can also set that environment variable to bogus externally so I removed the change.
![kafka-connect-http_error](https://user-images.githubusercontent.com/1239771/103158480-a1de7080-47be-11eb-82e2-0a721802ad74.png)